sim/druid/thrash.go

```diff
@@ -0,0 +1,87 @@
+package druid
+
+import (
+	"time"
+
+	"github.com/wowsims/cata/sim/core"
+)
+
+func (druid *Druid) registerThrashBearSpell() {
+	flatBaseDamage := 1042.0 // need variance?
+	flatBleedDamage := 581.0
```

Comment:
The DB entry for Thrash can be found here:
https://wago.tools/db2/SpellEffect?build=4.4.0.54377&filter[SpellID]=77758&page=1

Using this, I would recommend re-writing this code like I did the Ferocious Bite code so that it's clearer where the values are coming from:
```
// Raw parameters from spell database
hitCoefficient := 1.05599999428
hitVariance := 0.20999999344
bleedCoefficient := 0.58899998665

// Scaled parameters for spell code
avgBaseDamage := hitCoefficient * SpellScalingConstant // second factor is defined in druid.go
damageSpread := variance * avgBaseDamage
flatBaseDamage := avgBaseDamage - damageSpread / 2
flatBleedDamage := bleedCoefficient * SpellScalingConstant
```